src/main/java/frc/robot/subsystems/CubeapultSubsystem.java

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.PneumaticHub;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.StartEndCommand;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.robot.classes.DebugLog;
+
+public class CubeapultSubsystem extends SubsystemBase {
+  private static final int solenoidForwardID = 0; //TODO: Update ids when plugged in to the robot
+  private static final int solenoidReverseID = 1; //TODO: Update ids when plugged in to the robot
+  private static final int pneumaticsID = 59;
+  private DoubleSolenoid solenoid;
+  private PneumaticHub pneumatics;
+  private ShuffleboardTab configTab = Shuffleboard.getTab("Config");
+  private DebugLog<Boolean> solenoidStateLog = new DebugLog<Boolean>(false, "Solenoid State", this);
+  
+  public CubeapultSubsystem() {
+    configTab.add(new StartEndCommand(this::activate, this::deactivate, this).withName("Enable Compressor"));
+    solenoid = new DoubleSolenoid(PneumaticsModuleType.REVPH, solenoidForwardID, solenoidReverseID);
+    pneumatics = new PneumaticHub(pneumaticsID);
+    retract();
+    deactivate();
```

Comment:
1. Will this work when the robot is disabled?
2. Activate is never called anywhere to re-enable the compresser